applications/StructuralMechanicsApplication/custom_response_functions/adjoint_schemes/adjoint_structural_static_scheme.h

```diff
@@ -75,13 +75,16 @@ class AdjointStructuralStaticScheme : public Scheme<TSparseSpace, TDenseSpace>
 
         Parameters default_params(R"(
         {
-            "scheme_type": "adjoint_structural"
+            "scheme_type": "adjoint_structural",
+            "rotation_dofs": false
```

Comment:
please also remove this, I am not sure what happens if the setting is already present

Reply:
The setting is added to the parameter in python before it is given to this cpp constructor.
So it has to stay in the defaults here, otherwise it would give an error in the validation.

Reply:
ah sorry, I thought this was in python